### PR TITLE
Docs/acl changes admin partitions 1.11.x

### PR DESCRIPTION
**Pull request by trujillo-adam**
This change updates the ACL System docs to include information about admin partitions.
I implemented a few additional improvements, but we will need to iterate on the content to make it more usable.

### CODE REVIEW

File: website/content/docs/security/acl/acl-system.mdx [hunk 1]
```diff
@@ -69,16 +69,53 @@ If the ACL system becomes inoperable, you can follow the
...
 
-An ACL policy is a named set of rules and is composed of the following elements:
-
-- **ID** - The policy's auto-generated public identifier.
-- **Name** - A unique meaningful name for the policy.
-- **Description** - A human readable description of the policy. (Optional)
-- **Rules** - Set of rules granting or denying permissions. See the [Rule Specification](/docs/acl/acl-rules#rule-specification) documentation for more details.
-- **Datacenters** - A list of datacenters the policy is valid within.
-- **Namespace** - <EnterpriseAlert inline /> - The namespace this policy resides within. (Added in Consul Enterprise 1.7.0)
+An ACL policy (not to be confused with [policy dispositions](/docs/security/acl/acl-rules#policy-dispositions)) is a named set of rules and several attributes that define the policy domain. The ID is generated when policy is created, but you can specify the attributes when creating the policy. Refer to the [ACL policy command line](https://www.consul.io/commands/acl/policy) documentation or [ACL policy API](/api-docs/acl/policies) documentation for additional information on how to create policies.
```
Comment (karl-cardenas-coding) [inline website/content/docs/security/acl/acl-system.mdx:72]:
```suggestion
An ACL policy (not to be confused with [policy dispositions](/docs/security/acl/acl-rules#policy-dispositions)) is a named set of rules and several attributes that define the policy domain. The ID is generated when the policy is created, but you can specify the attributes when creating the policy. Refer to the [ACL policy command line](https://www.consul.io/commands/acl/policy) documentation or [ACL policy API](/api-docs/acl/policies) documentation for additional information on how to create policies.
```

**Review by karl-cardenas-coding — APPROVED** [review]
lgtm

File: website/content/docs/security/acl/acl-system.mdx [hunk 2]
```diff
@@ -179,26 +215,66 @@ of the following elements:
 - **Service Identity Set** - The list of service identities that are applicable for the role.
 - **Namespace** <EnterpriseAlert inline /> - The namespace this policy resides within. (Added in Consul Enterprise 1.7.0)
 
--> **Consul Enterprise Namespacing** - Roles may only link to policies defined in the same namespace as the role itself.
+-> **Linking Roles to Policies in Consul Enterprise** - Roles can only be linked to policies that are defined in the same namespace or admin partition.
```
Comment (freddygv) [inline website/content/docs/security/acl/acl-system.mdx:218]: Both the namespace and partition must match:
```suggestion
-> **Linking Roles to Policies in Consul Enterprise** - Roles can only be linked to policies that are defined in the same namespace and admin partition. 
```

File: website/content/docs/security/acl/acl-system.mdx [hunk 3]
```diff
@@ -179,26 +215,66 @@ of the following elements:
...
 - **Namespace** <EnterpriseAlert inline /> - The namespace this policy resides within. (Added in Consul Enterprise 1.7.0)
 
--> **Consul Enterprise Namespacing** - Roles may only link to policies defined in the same namespace as the role itself.
+-> **Linking Roles to Policies in Consul Enterprise** - Roles can only be linked to policies that are defined in the same namespace or admin partition. 
 
 ### ACL Tokens
 
-ACL tokens are used to determine if the caller is authorized to perform an action. An ACL token is composed of the following
-elements:
+Consul uses ACL tokens to determine if the caller is authorized to perform an action. An ACL token is composed of several attributes that you can specify when creating the token. Refer to the [ACL token command line](https://www.consul.io/commands/acl/token) documentation or [ACL token API](/api-docs/acl/tokens) documentation for additional information on how to create policies.:
```
Comment (freddygv) [inline website/content/docs/security/acl/acl-system.mdx:222]: There are some references to policies below that I think should instead be to tokens:
```suggestion
Consul uses ACL tokens to determine if the caller is authorized to perform an action. An ACL token is composed of several attributes that you can specify when creating the token. Refer to the [ACL token command line](https://www.consul.io/commands/acl/token) documentation or [ACL token API](/api-docs/acl/tokens) documentation for additional information on how to create tokens.:
```

File: website/content/docs/security/acl/acl-system.mdx [hunk 4]
```diff
@@ -179,26 +215,66 @@ of the following elements:
...
-  the primary datacenter and globally replicated.
-- **Expiration Time** - The time at which this token is revoked. (Optional; Added in Consul 1.5.0)
-- **Namespace** <EnterpriseAlert inline /> - The namespace this policy resides within. (Added in Consul Enterprise 1.7.0)
+- **Role Set** - The list of roles that are applicable for the token. Added in Consul 1.5.0.
+- **Service Identity Set** - The list of service identities that are applicable for the token. Added in Consul 1.5.0
+- **Local** - Indicates whether the token is local to the datacenter in which it was created. The attribute also can specify if the token was created in the primary datacenter and globally replicated.
+- **CreateTime** - Timestamp indicating when the token was created. 
+- **Expiration Time** - The time at which this token is revoked. This attribute is option when creating a token. Added in Consul 1.5.0.
+- **Namespace** <EnterpriseAlert inline /> - The namespace in which the policy resides. Added in Consul Enterprise 1.7.0.
+- **Partition** <EnterpriseAlert inline /> - The partition in which the policy resides. Added in Consul Enterprise 1.11.0.
```
Comment (freddygv) [inline website/content/docs/security/acl/acl-system.mdx:233]:
```suggestion
- **Namespace** <EnterpriseAlert inline /> - The namespace in which the token resides. Added in Consul Enterprise 1.7.0.
- **Partition** <EnterpriseAlert inline /> - The partition in which the token resides. Added in Consul Enterprise 1.11.0.
```

File: website/content/docs/security/acl/acl-system.mdx [hunk 5]
```diff
@@ -179,26 +215,66 @@ of the following elements:
...
+
+-> **Linking Tokens to Policies in Consul Enterprise** - Tokens can only be linked to policies that are defined in the same namespace or admin partition.
```
Comment (freddygv) [inline website/content/docs/security/acl/acl-system.mdx:235]: Same as above:
```suggestion
-> **Linking Tokens to Policies in Consul Enterprise** - Tokens can only be linked to policies that are defined in the same namespace and admin partition. 
```

**Review by freddygv — COMMENTED** [review]
Thanks Adam!. I left some feedback on the ACL-related commit, will comment on the mesh gateway changes in #11859

**Comment by hc-github-team-consul-core** [comment]
:cherries: If backport labels were added before merging, cherry-picking will start automatically.

To retroactively trigger a backport after merging, add backport labels and re-run https://circleci.com/gh/hashicorp/consul/536054.

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit 15a9e608374a43e2a516f38fda911e5706f15deb onto `stable-website` succeeded!

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit 15a9e608374a43e2a516f38fda911e5706f15deb onto `release/1.11.x` succeeded!

**Pull request merged**